README.md

```diff
@@ -164,8 +164,9 @@ See [whatwg/html#4697](https://github.com/whatwg/html/issues/4697) for more disc
 - [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
 - `open`: a boolean attribute, determining whether the toast is visible or not (according to the default styles).
 By default toasts are not shown.
-- `theme`: one of `"default"`, ???, or ???, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided)
-    - TODO([#18](https://github.com/jackbsteinberg/std-toast/issues/18)): decide on list of themes to natively support and create and style them.
+- `type`: category of toast, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided). This will correspond closely with certain [WAI-ARIA roles](https://www.w3.org/TR/aria-in-html/#aria-roles) 
```

Comment:
```suggestion
- `type`: category of toast, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided). This will correspond closely with certain [WAI-ARIA roles](https://w3c.github.io/using-aria/#aria-roles) 
```

Always link to Editor's Drafts, not /TR documents, as Editor's Drafts are more recently updated.

Reply:
Noted